INFO/src/main/kotlin/HelloWorldFile.kt

```diff
@@ -0,0 +1,10 @@
+/**
+ * Created by artur on 05.08.2017.
+ */
+
+
+fun main (args: Array<String>) {
```

Comment:
Du fügst eine neue Datei hinzu, aber die alte (in `src/main/kotlin/org.archdtd111.INFO/HelloWorldFile.kt`) wurde nicht gelöscht. Falls die alte Datei bei dir nicht mehr vorhanden ist, musst du Git mitteilen, dass du sie gelöscht hast. Das geht ganz normal mit `git add` und dann `git commit`.